File: refactored/duplicate-assert/java-junit/TestCoordResumeXCommand_testCoordSuspendAndResumeForRunning.java
public class TestCoordResumeXCommand extends TestCase {
    public void testCoordSuspendAndResumeForRunning1() throws Exception {
        CoordinatorJobBean job = addRecordToCoordJobTable(CoordinatorJob.Status.RUNNING, false, false);

        JPAService jpaService = Services.get().get(JPAService.class);
        assertNotNull(jpaService);
        CoordJobGetJPAExecutor coordJobGetCmd = new CoordJobGetJPAExecutor(job.getId());
        job = jpaService.execute(coordJobGetCmd);
        assertEquals(job.getStatus(), CoordinatorJob.Status.RUNNING);

        new CoordSuspendXCommand(job.getId()).call();
        job = jpaService.execute(coordJobGetCmd);
        assertEquals(job.getStatus(), CoordinatorJob.Status.SUSPENDED);
    }

    public void testCoordSuspendAndResumeForRunning2() throws Exception {
        CoordinatorJobBean job = addRecordToCoordJobTable(CoordinatorJob.Status.RUNNING, false, false);

        JPAService jpaService = Services.get().get(JPAService.class);
        assertNotNull(jpaService);
        CoordJobGetJPAExecutor coordJobGetCmd = new CoordJobGetJPAExecutor(job.getId());
        job = jpaService.execute(coordJobGetCmd);

        new CoordSuspendXCommand(job.getId()).call();
        job = jpaService.execute(coordJobGetCmd);
        assertEquals(job.getStatus(), CoordinatorJob.Status.SUSPENDED);

        new CoordResumeXCommand(job.getId()).call();
        job = jpaService.execute(coordJobGetCmd);
        assertEquals(job.getStatus(), CoordinatorJob.Status.RUNNING);
    }
}

File: refactored/duplicate-assert/python-pytest/TestCoordResumeXCommand_testCoordSuspendAndResumeForRunning.py
import pytest

class TestCoordResumeXCommand:
    def test_coord_suspend_and_resume_for_running_1(self):
        job = add_record_to_coord_job_table(CoordinatorJob.Status.RUNNING, False, False)

        jpa_service = Services.get().get(JPAService)
        assert jpa_service is not None
        coord_job_get_cmd = CoordJobGetJPAExecutor(job.get_id())
        job = jpa_service.execute(coord_job_get_cmd)
        assert job.get_status() == CoordinatorJob.Status.RUNNING

        CoordSuspendXCommand(job.get_id()).call()
        job = jpa_service.execute(coord_job_get_cmd)
        assert job.get_status() == CoordinatorJob.Status.SUSPENDED

    def test_coord_suspend_and_resume_for_running_2(self):
        job = add_record_to_coord_job_table(CoordinatorJob.Status.RUNNING, False, False)

        jpa_service = Services.get().get(JPAService)
        assert jpa_service is not None
        coord_job_get_cmd = CoordJobGetJPAExecutor(job.get_id())
        job = jpa_service.execute(coord_job_get_cmd)

        CoordSuspendXCommand(job.get_id()).call()
        job = jpa_service.execute(coord_job_get_cmd)
        assert job.get_status() == CoordinatorJob.Status.SUSPENDED

        CoordResumeXCommand(job.get_id()).call()
        job = jpa_service.execute(coord_job_get_cmd)
        assert job.get_status() == CoordinatorJob.Status.RUNNING
